project/database/seeders/DatabaseSeeder.php

```diff
@@ -49,5 +64,93 @@ public function run()
             'address'=>'Szczęśliwa 7, 80-176 Gdańsk',
             'location'=> new Point(54.35313340101314, 18.521470337609315)
         ]);
+
+        Category::create(['name'=>'Cementy i zaprawy']);
+        Category::create(['name'=>'Izolacja']);
+
+        Product::create([
+            'name'=>'Zaprawa murarska Huzar 25 kg',
+            'category_id'=>1,
```

Comment:
Jeżeli potrzebujesz podać `id` to nie jako "magic constant" bo nie masz pewności czy baza na pewno nada ten id przy tworzeniu rekordu. Lepiej jest to przekazać jako wartość pola z utworzonego obiektu:
```php
$category1 = Category::create(['name'=>'Cementy i zaprawy']);
//...
Product::create([
            'name'=>'Zaprawa murarska Huzar 25 kg',
            'category_id' => $category->id,
//...
]);